README.md

```diff
@@ -1069,7 +1069,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [appsmith](https://www.appsmith.com/) — Low code project to build admin panels, internal tools, and dashboards. Integrates with 15+ databases and any API.
   * [ToolJet](https://www.tooljet.com/) — Extensible low-code framework for building business applications. Connect to databases, cloud storages, GraphQL, API endpoints, Airtable, etc and build apps using drag and drop application builder.
   * [ReTool](https://retool.com/) — Low-code platform for building internal applications. Retool is highly hackable. If you can write it with JavaScript and an API, you can build it in Retool. The free tier allows up to five users per month, unlimited apps and API connections.
-
+  * [DronaHQ](https://www.dronahq.com/) — DronaHQ - a low code platform helps engineering teams and product managers to build internal tools, custom user journeys, digital experiences, automations, custom admin panels, operational apps 10X faster. 
```

Comment:
pricing page is unclear if this is a free tier that runs for ever since the button says "start free trial" and nowhere does it explain what happens at the end of the trial.

Reply:
developer plan is forever free plan. After the trial of the starter plan, automatically we move the user to the developer plan, which is free forever.

Reply:
I need to see that in the description on the site or in the FAQ etc

Reply:
We have updated our FAQ, you can see the updated details in "What is included in the free trial?" FAQ

Reply:
It says its a 14 day trial, nothing about what happens after.

![Screenshot 2023-02-24 at 10 26 38](https://user-images.githubusercontent.com/82342/221142093-38883ab1-8a50-45d1-9f4e-3aebc063ad4e.png)

Reply:
Can you refresh and check? 
<img width="624" alt="Screenshot 2023-02-24 at 12 40 15 PM" src="https://user-images.githubusercontent.com/114397573/221156524-a34b3f60-f473-4a35-9fe9-0ae46cded884.png">

Reply:
same, even on a different browser. This is the FAQ on the pricing page.

Reply:
Can you try closing the browser and check again, we have updated from our side
<img width="832" alt="Screenshot 2023-02-24 160521" src="https://user-images.githubusercontent.com/114397573/221157694-8d5c0b51-86f8-485d-836f-ed5ca99ccf92.png">

Reply:
I know how to refresh a page.  As I mentioned I also used a DIFFERENT BROWSER.  Same with chrome, safari and firefox.  Not using any local network caches etc.

Reply:
I totally agree with you, Okay let me check the concern at my end, then I will get back to you with the update.

Reply:
Hey, can you check now, we have updated it again. 
<img width="959" alt="Screenshot 2023-02-24 174917" src="https://user-images.githubusercontent.com/114397573/221177485-d4a9d34d-b919-401e-a3d1-be42795331ad.png">